server/impl/src/main/java/com/walmartlabs/concord/server/org/secret/SecretUpdateParams.java

```diff
@@ -37,10 +38,7 @@ public interface SecretUpdateParams extends Serializable {
     String newOrgName();
 
     @Nullable
-    UUID newProjectId();
-
-    @Nullable
-    String newProjectName();
```

Comment:
These need to stick around for now. There isn't really a good reason someone (outside of our project) should be using it, but since it's `public` this is technically a breaking change.

Annotate with `@Deprecated` and we can remove them in a subsequent release (post-JDK17 lang level?)

Reply:
Added these attributes back with deprecated annotation.